src/internal/connector/exchange/api/events.go

```diff
@@ -99,6 +99,47 @@ func (c Events) GetContainerByID(
 	return graph.CalendarDisplayable{Calendarable: cal}, nil
 }
 
+// GetContainerByName fetches calendar by name.
+// Note: At the moment, we only support lookups for calendars which belong to
+// the default calendar group.
```

Comment:
This is a bit misleading, since we don't investigate calendar groups at all.  (and by default, all calendars are part of the default calendar group).  I think you could remove it, as that might produce less confusion overall.
```suggestion
```

Reply:
good point